recipe/meta.yaml

```
@@ -17,14 +17,20 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - cmake
-    - make
+    - make  # [not win]
     - ninja  # [win]
```

```suggestion
    - ninja  # [win]
    - libgomp # [linux and blas_impl != "mkl"]
    - llvm-openmp # [osx and blas_impl != "mkl"]
```
libgomp and llvm-openmp have strong run_exports, which means they have to be placed in build for it to be effective.